android/assets/jsons/translations/German.properties

```diff
@@ -1426,11 +1426,20 @@ Worst = Schlechtester
 Turns until the next\ndiplomacy victory vote: [amount] = Runden bis zur nächsten Abstimmung\nüber den Diplomatiesieg: [amount]
 Choose a civ to vote for = Wähle eine Zivilisation, für die du abstimmen möchtest
 Choose who should become the world leader and win a Diplomatic Victory! = Wähle, wer der Anführer der Welt werden und somit den Diplomatiesieg erhalten soll
-Voted for = Abgestimmt für
+Voted for = hat gestimmt für
 Vote for [civilizationName] = Abstimmen für [civilizationName]
+Abstain = Enthalten
 Continue = Fortfahren
 Abstained = Enthalten
 Vote for World Leader = Stimme für den Anführer der Welt ab
+[number] votes = [number] Stimmen
+[number] vote = [number] Stimme
```

Comment:
If this is always going to be 1 we don't need the brackets
Alternatively we could do "[amount] vote(s)"

Reply:
...they can use a non-breaking space character, that should technically work, interesting

Reply:
No you're right, hm
I got nothing
But no one has yet requested that, so I don't think we need to solve that problem if it's not a problem :)